packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -3724,6 +3724,53 @@ void main() {
       },
     );
 
+    testWidgets('menus should respect overflow on right with the provided offset', (
+      WidgetTester tester,
+    ) async {
+      await changeSurfaceSize(tester, const Size(800, 600));
+      await tester.pumpWidget(
+        MaterialApp(
+          theme: ThemeData(useMaterial3: false),
+          home: Builder(
+            builder: (BuildContext context) {
+              return Directionality(
+                textDirection: TextDirection.ltr,
```

Comment:
Directionality is provided by `MaterialApp`, this can be skipped.

Reply:
Sure